runtime/src/server/middleware/get_page_handler.ts

```diff
@@ -109,13 +109,14 @@ export function get_page_handler(
 				preload_error = { statusCode, message };
 			},
 			fetch: (url: string, opts?: any) => {
-				const parsed = new URL.URL(url, `http://127.0.0.1:${process.env.PORT}${req.baseUrl ? req.baseUrl + '/' :''}`);
+				const protocol = req.socket.encrypted ? 'https' : 'http';
```

Comment:
please drop the changes from https://github.com/sveltejs/sapper/pull/1358 from this PR

Reply:
Yes, I will do that